src/components/contact/ContactIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import { Text, Box } from '@mantine/core';
+import { IconSun} from '@tabler/icons-react';
+import './ContactIcons.scss';
+
+interface ContactIconProps
```

Comment:
why the props aren't exported. they never used ?

Reply:
Never used so don't need to export